packages/datagateway-dataview/src/page/pageContainer.component.tsx

```diff
@@ -149,6 +150,9 @@ const NavBar = React.memo(
   }): React.ReactElement => {
     const [t] = useTranslation();
 
+    //Determine whether logged in anonymously
+    const loggedInAnonymously = localStorage.getItem('autoLogin') === 'true';
```

Comment:
A user can technically login as anonymous and not be `autoLogin === 'true'` - perhaps it's better to check that the username contains "anon" instead? (as the current anon username by default is `anon/anon`)

Reply:
Is there any reason not to require the username is exactly 'anon/anon', or maybe 'anon/' to avoid it falsely recognising someone with anon in their username?

Reply:
Theoretically `authn.anon` could be setup with no mnemonic or a different mnemonic other than `anon`, meaning the anon username could be just `anon`, or `anonymous/anon`, or `unauthenticated/anon` etc. However this is a very specific problem and I don't think anyone ever configures `authn.anon` that way and so the default is pretty much always `anon/anon`.

However, I think in SciGateway we assume the anonymous authenticator is setup with `anon` as the mnemonic, so maybe it's fine and consistent to expect `anon/anon`

Reply:
@louise-davies Okay, I have changed it to that for now in the latest commit.